packages/dataviews/src/components/dataviews/index.tsx

```diff
@@ -193,6 +195,40 @@ function DataViews< Item >( {
 		}
 	}, [ hasPrimaryOrLockedFilters, isShowingFilter ] );
 
+	const isInfiniteScroll = useMemo( () => {
+		if ( ! infiniteScrollHandler ) {
+			return false;
+		}
+		return !! view.layout?.infiniteScroll;
+	}, [ view, infiniteScrollHandler ] );
```

Comment:
I think you could remove the `useMemo` here, and instead have the boolean calculated inline.

Reply:
We can if we assume that `view.layout?.infiniteScroll` will only exist if the `infiniteScrollHandler` also exists. Which it should! I might have been overly cautious here.